Makefile

```diff
@@ -15,12 +15,15 @@ $(TARGETS): .dapper
 shell: .dapper
 	./.dapper -s -m bind
 
-bin/subctl:  pkg/subctl/operator/install/embeddedyamls/yamls.go $(shell find pkg/subctl/ -name "*.go")
+bin/subctl: pkg/subctl/operator/install/embeddedyamls/yamls.go pkg/subctl/lighthouse/install/embeddedyamls/yamls.go $(shell find pkg/subctl/ -name "*.go")
```

Comment:
Why not adding them all on the same embedded yamls file?

It's in the end operator yamls...

Having separate files make's it more complex and buys little. I don't want to slow you down at this moment, but if we can put them back together on a later PR, I feel that's easier to maintain.  (and of course, not adding extra ones for coastguard later on)

Reply:
That is what Stephen is doing in his patch - https://github.com/submariner-io/submariner-operator/pull/129

It was either I wait for Stephen's refactoring to be merged, or we merge this and Stephen merges his. This being smaller patch felt like better choice to go in first.